src/Renci.SshNet/Security/Cryptography/EcdsaDigitalSignature.cs

```diff
@@ -39,7 +39,7 @@ public override bool Verify(byte[] input, byte[] signature)
             // for 521 sig_size is 132
             var sig_size = _key.KeyLength == 521 ? 132 : _key.KeyLength / 4;
             var ssh_data = new SshDataSignature(signature, sig_size);
-#if NETSTANDARD2_0
+#if NETSTANDARD2_0 || NETSTANDARD2_1
```

Comment:
Please use ```#if NETSTANDARD2_0_OR_GREATER || NETCOREAPP3_1_OR_GREATER``` in all instances, so this will be compiled when we target higher frameworks.

Reply:
certainly I can make such adjustment